pkg/sql/logictest/testdata/logic_test/distsql_stats

```diff
@@ -2794,16 +2794,58 @@ statement ok
 INSERT INTO mno (m, n) SELECT i, i % 50 FROM generate_series(0, 999) s(i)
 
 statement ok
-ANALYZE mno
+CREATE STATISTICS virt FROM mno
 
-query TTIIB
+query TTIIB colnames
 SELECT statistics_name, column_names, row_count, distinct_count, histogram_id IS NOT NULL AS has_histogram
 FROM [SHOW STATISTICS FOR TABLE mno]
 ORDER BY statistics_name, column_names::STRING
 ----
-NULL  {m}  1000  1000  true
-NULL  {n}  1000  50    true
-NULL  {o}  1000  33    true
+statistics_name  column_names  row_count  distinct_count  has_histogram
+virt             {m}           1000       1000            true
+virt             {n}           1000       50              true
+virt             {o}           1000       33              true
+
+let $hist_o
+SELECT histogram_id FROM [SHOW STATISTICS FOR TABLE mno]
```

Comment:
I'm somewhat sceptical that this addition will help in understanding the failure. What are the reasons for "missing stats" during the planning? We already checked in the query above that there are some stats on columns `m`, `n`, and `o`, so the stats do get collected and written down into the system table, and we're just not using them during planning for some reason. If the histogram for `o` got busted, then I'd have expected that we'd still use stats on `n`.

nit: also EXPLAIN plans should be in execbuilder tests.

Reply:
Good points. I'll close this.